lib/landable/traffic/user_tracker.rb

```diff
@@ -38,8 +38,17 @@ def record_page_view
         end
       end
 
+      def amqp_config_hash
+        @amqp_config_hash = Landable.configuration.amqp_configuration
+      end
+
       def save
-        record_page_view
+        p = record_page_view
+        d = PageView.find(p.page_view_id).created_at
+        if amqp_config_hash[:enabled] \
+              && amqp_config_hash[:messaging_service].present?
```

Comment:
we are using this logic multiple times, maybe we just bring it to the [configuration object](https://github.com/gevansBigG/landable/blob/amqp_via_bunny/lib/landable/configuration.rb)

and in there 
```ruby
def ampq_enabled?
  Landable.configuration.amqp_configuration[:enabled] && Landable.configuration.amqp_configuration[:messaging_service].present?
end
```
Then you can just call ```Landable.configuration.ampq_enabled? ```